ExomeDepth/submit_batch_exomedepth.py

```diff
@@ -72,31 +73,16 @@ def exomedepth_analysis(bam, args, gender_dic, suffix_dic, refset_dic):
     os.system("mv {output}/{sampleid}/UMCU*.vcf {output}/UMCU/".format(sampleid=sampleid, output=args.outputfolder))
     os.system("rm -r {output}/{sample}".format(output=args.outputfolder, sample=sampleid))
 
-    return [sampleid, bamfile, refset]
-
-
-def parse_ped(ped_file):
-    samples = {}  # 'sample_id': {'family': 'fam_id', 'parents': ['sample_id', 'sample_id']}
-    for line in ped_file:
-        ped_data = line.strip().split()
-        family, sample, father, mother, sex, phenotype = ped_data
-
-        # Create samples
-        if sample not in samples:
-            samples[sample] = {'family': family, 'parents': [], 'children': []}
-        if father != '0' and father not in samples:
-            samples[father] = {'family': family, 'parents': [], 'children': []}
-        if mother != '0' and mother not in samples:
-            samples[mother] = {'family': family, 'parents': [], 'children': []}
+    hc_cnv_vcf = f"{args.outputfolder}/HC/HC_{refset}_{sampleid}_{run}_exome_calls"
+    ed_igv = f"{args.outputfolder}/igv_tracks/HC_{refset}_{sampleid}_{run}"
+    if sampleid in suffix_dic:
+        hc_cnv_vcf = "{}_{}.vcf".format(hc_cnv_vcf, suffix_dic[sampleid])
+        ed_igv = "{}_{}_ref.igv".format(ed_igv, suffix_dic[sampleid])
+    else:
+        hc_cnv_vcf = f"{hc_cnv_vcf}.vcf"
+        ed_igv = f"{ed_igv}_ref.igv"
```

Comment:
Make a choice between f-string or .format

Reply:
Applied f-string in entire function